pkg/database/plugin.go

```diff
@@ -459,7 +471,8 @@ const (
 type HashCollectionNS CollectionName
 
 const (
-	CollectionGroups HashCollectionNS = "groups"
+	CollectionGroups        HashCollectionNS = "groups"
+	CollectionTokenAccounts HashCollectionNS = "tokenaccounts"
```

Comment:
I chose `HashCollectionNS` (as the token account should be identifiable by the combination of pool+tokenIndex+identity). Want to be sure that's correct though, since we don't have a lot of things that fall into this bucket.

Reply:
aha, this did indeed answer my earlier questions.

Yes, I think this is consistent with what we do for groups. It's slightly different, as for Groups we actually need to encode a variable size set of data together for uniqueness (a list of members), where as here it's always three strings. So I think we're saying the choice on the REST API is between:

`GET` `namespaces/{ns}/tokens/{type}/pools/{name}/accounts/{hash}`

and

`GET``namespaces/{ns}/tokens/{type}/pools/{name}/accounts/{index}/{identity}`

I guess my gut says the latter would be more obvious to people, but on the other hand `identity` in this context moving to instead being a `signing key` (because here we don't actually know the identity) expressed as a URI (per https://github.com/hyperledger-labs/firefly/issues/187). And having a URI within a pay in a URI would be wonky.

So the conclusion to use a `hash` sounds fine - if you want to query by identity/index you would need to use a custom filter query. Which we support (and would probably need indexes for in an upcoming PR)

Reply:
Yea, I was leaning more toward the latter. I couldn't decide if `{index}` actually belongs before or after `accounts` in the URI, and wasn't sure how `{identity}` should be represented in the URI (and if it should even be in the path at all, or should be a query param).